### PR TITLE
Beginnings of the revamp

### PR DESCRIPTION
**Pull request by kieranjarrett**
The starting of updating the documentation to Jeeves 4.0

### CODE REVIEW

File: docs/commands/dashboard/role-management.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+|Only Demote|This option means that Jeeves will only remove roles from the user and will not add new roles that apply to the user.|
+
+### Mode
+
+The Mode drop down allows you to decide if Jeeves should automatically update the roles of a player or if the user needs to run `!toon update`. 
+
+|Option|Description|
+|----------|-----------|
+|Auto|This means that Jeeves will apply the roles when a player runs `!auth`, when a player joins the server and has already run `!auth` on another server, or when a player runs `!toon update`.|
+|Manual|This means that Jeeves will only apply roles when a player runs `!toon update`.|
```
Comment (SirDeadlystrike) [inline docs/commands/dashboard/role-management.md]: This should be `!role update` not `!toon update`

File: docs/commands/dashboard/role-management.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+|Only Promote|This option means that Jeeves will only add roles to the user and will not remove those that no longer apply.|
+|Only Demote|This option means that Jeeves will only remove roles from the user and will not add new roles that apply to the user.|
+
+### Mode
+
+The Mode drop down allows you to decide if Jeeves should automatically update the roles of a player or if the user needs to run `!toon update`. 
+
+|Option|Description|
+|----------|-----------|
+|Auto|This means that Jeeves will apply the roles when a player runs `!auth`, when a player joins the server and has already run `!auth` on another server, or when a player runs `!toon update`.|
```
Comment (SirDeadlystrike) [inline docs/commands/dashboard/role-management.md]: I would add some more clarification about auto vs manual.

EX:
Auto - Jeeves will where applicable automatically update roles for users on certain events. Such as....
- A user linking/updating their battlenet account via `!auth`
- A user forcing an update of their toons via `!toon update`
- Jeeves Premium Role Automation

Manual - Jeeves will only ever update a users roles if they run the `!role update` command in your server

File: mkdocs.yml [hunk 3]
```diff
@@ -5,16 +5,24 @@ extra_css:
...
-        - 'Bot Channel': 'commands/Setup/bot-channel.md'
-        - 'Greeting': 'commands/Setup/greeting.md'
-        - 'Guild': 'commands/Setup/guild.md'
-        - 'Guild Roles': 'commands/Setup/guild-roles.md'
-        - 'Prefix': 'commands/Setup/setup-prefix.md'
-        - 'Realm': 'commands/Setup/Realm.md'
-        - 'Role Channel': 'commands/Setup/role-channel.md'
-        - 'Role Manager': 'commands/Setup/role-manager.md'
-        - 'Time zone': 'commands/Setup/timezone.md'
+    - Dashboard:
```
Comment (SirDeadlystrike) [inline mkdocs.yml]: This should probably not be in the Command Scope but instead given a top level spot or put under a new "Configuring Jeeves" scope

**Review by SirDeadlystrike — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 3 above)
Comment (kieranjarrett) [reply mkdocs.yml]: Shifted this in new commits. Looking at the guides menu, I think some could probably move into Configuration, some probably aren't needed any more, and then maybe a new menu for the Change log and the supporting Jeeves page?

**Comment by kieranjarrett** [comment]
@SirDeadlystrike  Need you to take a look at the Supporting Jeeves page I just drafted.

**Comment by kieranjarrett** [comment]
@SirDeadlystrike Do you prefer the written permissions style or the table? Or should the table include different functions and the permissions they require?


Also need you to update the roles per guild rank on the Supporting Jeeves page.

**Review by akintis008 — APPROVED** [review]

**Review by SirDeadlystrike — APPROVED** [review]

**Pull request merged**